bin/lib/check_reqs.js

```diff
@@ -78,21 +79,46 @@ module.exports.check_ant = function() {
     });
 };
 
+module.exports.get_gradle_wrapper = function() {
+    var androidStudioPath;
+    if(os.platform() == "darwin") {
```

Comment:
I have no idea where Android Studio is installed on a Linux box.  It literally could be anywhere!  If you're not using Android Studio, you HAVE to have Gradle installed.  It's either one or the other from here on out.